JS_lesson4/Task2.js

```diff
@@ -0,0 +1,31 @@
+// 1. Реализовать Task 1 через switch
+// 2. Преобразовать написанный код в task 1 так, чтобы сначала проверялся тип данных. 
+//   И если он не number - кидалась ошибка в консоль.
+//   Проверить работу кода на следующих данных 17, 18, 61, "2", "aaa"
+// 3. Преобразовать Task 2 - 1 таким образом, чтобы значение НАПРИМЕР '2' (т.е. ЛЮБАЯ строка в которой лежат ТОЛЬКО ЦИФРЫ) пропускалось, 
+//   преобразовываясь в number
+const age = "54";
+const age_2 = 18;
+const age_3 = 60;
+    switch (true) {
+        case Number.isNaN(Number(age)): {
+            throw new Error('Parameter is not a number!');
+            break;
+        } 
+        case Number(age) < age_2: {
+            console.log(`You dont have access cause your age is ${age} Its less then`);
+            break;
+        }
+        case Number(age) >= age_2 && Number(age) < age_3: {
+            console.log(`Welcome  !`);
+            break;
+        }
+        case Number(age) > age_3: { 
+            console.log(`Keep calm and look Culture channel`);
+            break;
+        }
+        default: {
+            console.log("Technical work")
+            break; 
+        }
+    }
```

Comment:
What is the purpose of declaring a string value in the `age` constant and then converting it to a number?

Pay attention to the description of the 2nd task - we only need to check the type of the value, if the value type is not a number, throw an error

Please provide a solution to the 2nd problem using the `if...else` statement.

Add a separate solution for the 3rd problem